cnxepub/models.py

```diff
@@ -128,12 +128,12 @@ def model_to_tree(model, title=None, lucent_id=TRANSLUCENT_BINDER_ID):
         {'id': <id>|'subcol', 'title': <title>, 'contents': [<tree>, ...]}
 
     """
-    if type(model) is TranslucentBinder:
+    id = model.ident_hash
+    if id is None and isinstance(model, TranslucentBinder):
```

Comment:
Given a non-identified binder (i.e. `binder.id is None`), the resulting id in the tree would be `subcol`. `type(model)` was used here because we want to know the exact type of the instance. The evaluation for this looks something like:
```python
assert isinstance(binder, TranslucentBinder)  # ✅ 
assert isinstance(translucent_binder, TranslucentBinder)  # ✅ 
assert isinstance(document, TranslucentBinder)  # ❌ 
```